goose3/extractors/authors.py

```diff
@@ -69,9 +69,9 @@ def __get_authors_from_schema(self):
                 schema_authors = [schema_authors]
             for author in schema_authors:
                 if isinstance(author, dict):
-                    author = author.get("name", None)
+                    extracted_author = author.get("name", None)
                     if author:
```

Comment:
Shouldn't this also be extracted author? Otherwise a `None` might be inserted into the array.

Reply:
You are completely right - stupid oversight on my part. Thanks for catching it!